### PR TITLE
75 query uisp in pure python and enhance output

### LINKED ISSUE #75: Query UISP in pure python and enhance output
We should be able to get the UISP data using an HTTP GET request to the right endpoint with the `requests ` library. Simplifies a lot if we can get it working

### PR DESCRIPTION
**Pull request by andybaumgar**

### CODE REVIEW

File: Dockerfile [hunk 1]
```diff
@@ -1,11 +1,7 @@
 # syntax=docker/dockerfile:1
 
-FROM byxorna/nycmesh-tool:latest AS mesh-tool
-
```
Comment (Andrew-Dickinson) [inline Dockerfile:4]: Nice!

File: tests/test_diagnostics_report.py [hunk 2]
```diff
@@ -0,0 +1,25 @@
+import os
+from dotenv import load_dotenv
+from supportbot.utils.diagnostics_report import get_report, lbe_traceroute_report
+
+load_dotenv()
+
+# commented out so test doesn't fail off Mesh
```
Comment (Andrew-Dickinson) [inline tests/test_diagnostics_report.py]: Maybe we could change this so it's a conditional check? Shouldn't be that hard to just wrap things in a try-execept?

File: supportbot/supportbot/utils/uisp_data.py [hunk 3]
```diff
@@ -0,0 +1,48 @@
...
+from dotenv import load_dotenv
+import os
+import re
+from datetime import datetime
+
+load_dotenv()
+
+def get_uisp_devices():
+    response = requests.get("https://uisp.mesh.nycmesh.net/nms/api/v2.1/devices", headers={'x-auth-token': os.environ.get('NYCMESH_TOOL_AUTH_TOKEN')}, verify=False)
+
```
Comment (Andrew-Dickinson) [inline supportbot/supportbot/utils/uisp_data.py:12]: nit: Might be good to pull this URL out into a separate constants file?

**Review by Andrew-Dickinson — APPROVED** [review]

File: tests/test_diagnostics_report.py (hunk 2 above)
Comment (andybaumgar) [reply tests/test_diagnostics_report.py]: I think ideally we would have a test server that is able to run the tests.

I guess in the mean time we could just use a ping to the UISP server as the condition.

File: supportbot/supportbot/utils/uisp_data.py (hunk 3 above)
Comment (andybaumgar) [reply supportbot/supportbot/utils/uisp_data.py:12]: Oh good idea

File: tests/test_diagnostics_report.py (hunk 2 above)
Comment (Andrew-Dickinson) [reply tests/test_diagnostics_report.py]: Could also use an environment variable that gets set in the build system but not on the dev's laptop

Comment (andybaumgar) [reply tests/test_diagnostics_report.py]: I ended up using `@pytest.mark.skipif`

**Pull request merged**